docs/src/standards/rfcs/rfc9576.md

```
@@ -0,0 +1,1675 @@
+
+
+### 3.1. Overview
+
+
+This section describes the typical interaction flow for Privacy Pass,
+as shown in Figure 1.
+
+
+## 1. A Client interacts with an Origin by sending a request or
+
+       otherwise interacting with the Origin in a way that triggers a
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)